src/domain/file/services/file-service.ts

```diff
@@ -40,6 +43,73 @@ export class FileService {
         }
     }
 
+    async handleMultipartChunk(
+        userId: string,
+        file: File,
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
+        chunk: any,
+        partNumber: number,
+        s3Stuff: PromiseResult<S3.CreateMultipartUploadOutput, AWSError>,
```

Comment:
Could this param name be a bit more descriptive?

Reply:
Solved